Gemfile

```diff
@@ -25,6 +25,7 @@ group :development, :test do
   gem 'launchy'
   gem 'pry-byebug'
   gem 'rspec-rails'
+  gem 'mutant-rspec'
```

Comment:
Sorry, my bad. I didn't notice that was the way we were doing it. I'll change it.